appeals/api/src/server/tests/appeals/expectation.js

```diff
@@ -9,25 +9,58 @@ import { formatLpaQuestionnaire } from '#endpoints/lpa-questionnaires/lpa-questi
  * @param {Appeal} appeal
  * @returns {SingleLPAQuestionnaireResponse}
  */
+// @ts-ignore
 export const baseExpectedLPAQuestionnaireResponse = (appeal) => ({
-	...formatLpaQuestionnaire(appeal),
+	...formatLpaQuestionnaire(appeal), // TODO: shouldn't be calling this from tests
```

Comment:
I don't think this TODO is clear.

Reply:
I've added more detail to clarify this - the purpose of this file/function is to generate the expected output for use in the tests, but it uses the formatting function which is part of what's being tested. In the scenario where a bug exists in the formatting function which changes the output, this wouldn't be caught by any tests using this expectation, because the issue would be reflected in the expectation the test is comparing against. Hopefully this makes sense!